parachain/runtime/testnet-kintsugi/src/lib.rs

```diff
@@ -1051,13 +1118,14 @@ construct_runtime! {
     {
         System: frame_system::{Pallet, Call, Storage, Config, Event<T>} = 0,
         Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 1,
-        Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>} = 2,
-        Utility: pallet_utility::{Pallet, Call, Event} = 3,
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>} = 4,
-        Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 5,
-        Preimage: pallet_preimage::{Pallet, Call, Storage, Event<T>} = 6,
-        Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 7,
-        Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 8,
+        Utility: pallet_utility::{Pallet, Call, Event} = 2,
```

Comment:
Why this change? If you really want this, it'll be a breaking change. Also, please mark the pr as breaking

Reply:
To match the Kintsugi runtime, otherwise the testnet will have different indices

Reply:
Do we want to consider these breaking changes also to the testnet runtimes?

Reply:
I'm not sure, but this pr is breaking regardless (assuming the `proxy` pallet has events)

Reply:
Updated the title